pkg/configuration/configuration_test.go

```diff
@@ -495,3 +495,149 @@ func (s *TestConfigurationSuite) TestGetNamespace() {
 		assert.Equal(s.T(), newVal, config.GetNamespace())
 	})
 }
+
+func (s *TestConfigurationSuite) TestVerificationEnabled() {
+	key := configuration.EnvPrefix + "_" + "VERIFICATION_ENABLED"
+	resetFunc := UnsetEnvVarAndRestore(s.T(), key)
+	defer resetFunc()
```

Comment:
Even if I understand the logic, I find it a bit weird that you don't use the `resetFunc` in the last sub test

Reply:
It is a bit weird, I copied this structure from previous tests though.  My guess is that the test-level one is there as a "catch-all" in case any of the subtests fail to call it.